test/unit/api_spec.js

```diff
@@ -979,6 +979,13 @@ describe('api', function() {
       expect(viewport.width).toEqual(1262.835);
       expect(viewport.height).toEqual(892.92);
     });
+    it('gets viewport respecting "dontFlip" argument', function () {
+      var flipped = page.getViewport(1.5, 135);
+      var notFlipped = page.getViewport(1.5, 135, true);
+
+      expect(flipped.transform[2]).toEqual(-notFlipped.transform[2]);
+      expect(flipped.transform[3]).toEqual(-notFlipped.transform[3]);
```

Comment:
Perhaps a bit clearer, and more complete test coverage, if you explicitly compare against the expected values (similar to what's done in the test above):
```javascript
expect(flipped.transform).toEqual([/* values here... */]);
expect(notFlipped.transform).toEqual([/* values here... */]);
```

Reply:
ok
- [ ] compare whole transform arrays